openeogeotrellis/integrations/s3/bucket_details.py

```diff
@@ -19,6 +18,8 @@ class BucketDetails:
 
     @classmethod
     def from_name(cls, bucket_name: str) -> BucketDetails:
+        from openeogeotrellis.workspace.object_storage_workspace import ObjectStorageWorkspace
```

Comment:
This is to avoid circular dependency and to avoid having to use s3_client as module name #1190